apps/src/templates/ContentContainer.jsx

```diff
@@ -222,4 +224,9 @@ class Link extends Component {
   }
 }
 
-export default Radium(ContentContainer);
+export const UnconnectedContentContainer = ContentContainer;
+
+export default connect(state => ({
+  responsiveSize: state.responsive.responsiveSize,
+  isRtl: state.isRtl,
+}))(Radium(ContentContainer));
```

Comment:
You probably want `Radium` applied to both the connected and unconnected version of this component.  I recommend the following pattern:
```jsx
export const UnconnectedContentContainer = Radium(ContentContainer);

export default connect(state => ({
  // ...whatever...
}))(UnconnectedContentContainer);
```